auth_session_timeout/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,14 @@
+<!--
+    (c) 2015 ACSONE SA/NV, Dhinesh D
+    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+-->
+<odoo noupdate="1">
+    <record id="inactive_session_time_out_delay" model="ir.config_parameter">
+        <field name="key">inactive_session_time_out_delay</field>
+        <field name="value">7200</field>
+    </record>
+    <record id="inactive_session_time_out_ignored_url" model="ir.config_parameter">
+        <field name="key">inactive_session_time_out_ignored_url</field>
+        <field name="value">/calendar/notify,/longpolling/poll</field>
```

Comment:
I think this URL must be update to V16 for `web socket` instead of `longpolling`, what do you think?
Please see
https://github.com/odoo/odoo/blob/16.0/addons/bus/controllers/websocket.py#L13

Reply:
Good Catch!
Will update the PR.
Related: https://github.com/odoo/odoo/pull/75510
https://github.com/odoo/odoo/commit/de6de48deb983be910876c6133c2869907c142f7